package.json

```diff
@@ -18,53 +18,23 @@
     "crypto"
   ],
   "scripts": {
-    "prebuild": "rm -rf lib && rm -rf dist",
-    "build": "yarn run build:node && yarn run build:web && yarn run build:types",
-    "bundle": "webpack --config webpack.bundle.config.js",
-    "build:node": "babel --no-babelrc --config-file ./babel.config.json src --copy-files -x .ts -d dist --source-maps",
-    "build:web": "babel --no-babelrc --config-file ./babel.web.config.json src --copy-files -x .ts -d lib --source-maps",
-    "build:types": "tsc --declaration --outDir dist --emitDeclarationOnly",
-    "proto:gen": "pbjs -t static-module -o ./src/proto/payload.js ./src/proto/payload.proto && pbts -o ./src/proto/payload.d.ts ./src/proto/payload.js && yarn run lint --fix",
-    "check-types": "tsc --noEmit",
-    "lint": "eslint --ext .ts src/",
+    "build": "aegir build --ts",
+    "lint": "aegir lint --ts",
     "pretest": "yarn check-types",
```

Comment:
since build is ran using tsc (it will check types also), we don't need this to do type checking here